### PR TITLE
Lora in readme

### PR DESCRIPTION
**Pull request by rexfue**
LoRa description added
replaced a few double spaces with '\' to have a line break

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+ * Erste Zeile, Sensor ID:\
  Hier die Nummer (nur die Zahlen) der SSID des Sensors eingeben (z.B. bei dem Sensor ESP-51564452 also dann nur 51564452 eingeben)
- * Zweite Zeile, Sensor Board:
+ * Zweite Zeile, Sensor Board:\
  Hier *esp32* auswählen (über die kleinen Pfeile rechts)
- * Basisinformation:
+ * Basisinformation:\
  Hier die Adresse eingeben (mit dem Land!). Der interne Name des Sensors kann beliebig vergeben werden, muss aber eingegeben werden. Bitte den Haken bei **Indoor-Sensor** setzen, so lange der Sensor wirklich innen ist.
- * Zusätzliche Informationen:
+ * Zusätzliche Informationen:\
```
Comment (ThomasWaldmann) [inline README.md:171]: these `\` are somehow ugly. is there another way to solve this?

File: README.md [hunk 2]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+* **App Key:**\
+Keine Eingabe nötig
+* **App EUI:**\
+Bleibt so stehen
+
+Nun rechts unten auf **Register**. Nun ist auch das Device angelegt.
+
+### Übernahme der LoRa-Konstanten 
+Die LoRa-Konstanten müssen noch in das Programm übernommen werden.\
+Dazu muss die Konfigurations-Webseite des Geigerzählers aufgerufen werden. Weiter oben ist erklärt, wie die Konfiguartions-Seite zu erreichen ist (*Einstellung des WLAN* und *Aufruf aus dem WLAN*))
```
Comment (ThomasWaldmann) [inline README.md:225]: Typo Konfigurations-Seite

File: README.md [hunk 3]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+* **Device EUI:**\
+Einmal auf das Symbol ganz links an der Zeile klicken, dann erscheint der Text, dass diese Nummer vom System erzeugt wird. Wir müssen sonst nichts weiter eingeben.
+* **App Key:**\
+Keine Eingabe nötig
+* **App EUI:**\
+Bleibt so stehen
+
+Nun rechts unten auf **Register**. Nun ist auch das Device angelegt.
+
+### Übernahme der LoRa-Konstanten
```
Comment (ThomasWaldmann) [inline README.md:223]: s/Konstanten/Parameter/ (please do a global review / change for this)

File: README.md [hunk 4]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
- * Position
+ * Position\
  Hier bitte die Koordinaten eingeben, so genau wie möglich (oder über den rechten Knopf die Koordinaten rechnen lassen). Dies wird benötigt, um den Sensor später auf der Karte anzeigen zu können.
 
- Nun mit *Einstellungen speichern* das Ganze beenden. Dann auf der Übersichts-Seite bei diesem Sensor auf *Daten* klicken. Nun steht hinter *Sensor ID* die ID des Sensors. Diese bitte merken: sie wird für die Abfrage bei sensor.community bzw. bei der Anzeige auf https://multigeiger.citysensor.de benötigt (zur Zeit ist das noch laufend in Arbeit - kann/wird also ab und zu ausfallen :wink: ).
+ Nun mit *Einstellungen speichern* das Ganze beenden. Dann auf der Übersichts-Seite bei diesem Sensor auf *Daten* klicken. Nun steht hinter *Sensor ID* die ID des Sensors. Diese bitte merken: sie wird für die Abfrage bei sensor.community bzw. bei der Anzeige auf https://multigeiger.citysensor.de benötigt.
+ 
+## LoRa-Interface
+Um den Multigeiger in das TTN-Netz zu bekommen, sind ein paar vorbereitende Schritte durchzuführen:
```
Comment (ThomasWaldmann) [inline README.md:181]: an TTN ("The Things Network") anzubinden

File: README.md [hunk 5]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
 
+* Anlegen des TTN-Devices bei *The Things Network*
+* Übernehmen der Konstanten in den Multigeiger
```
Comment (ThomasWaldmann) [inline README.md:184]: Parameter

File: README.md [hunk 6]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+* Anmelden bei *sensor.community* (ehemals luftdaten.info)
+* HTTP-Integration
+
+
+### Anlegen eines TTN-Devices
+Das Gerät muss bei TTN (The Things Network) angemeldet werden. Dazu muss zuerst (falls noch nicht vorhanden) ein Account bei TTN amgelegt werden.
```
Comment (ThomasWaldmann) [inline README.md:190]: remove "(The Things Network)"

Comment (ThomasWaldmann) [reply README.md:190]: typo angelegt

File: README.md [hunk 7]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+
+#### Account anlegen
+Über <https://account.thethingsnetwork.org/register> auf die Webseite zum Account anlegen gehen. Hier dann einen **USERNAME** vergeben, die **EMAIL ADRESSE** eintragen und auch ein **PASSWORD** vergeben. Dann rechts unten über **Create account** den Account anlegen. Danach kann man sich mit den neuen Daten an der Console anmelden (<https://account.thethingsnetwork.org/users/login>).
```
Comment (ThomasWaldmann) [inline README.md:193]: **EMAIL ADDRESS**

File: README.md [hunk 8]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+
+#### Applikation anlegen
+Ist man eingelogged, wird über **APPLICATIONS** und **add applications** die neue Applikation angelegt. Folgende Felder müssen ausgefüllt werden:
```
Comment (ThomasWaldmann) [inline README.md:196]: Ist man angemeldet, ...

File: README.md [hunk 9]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+
+* **Application ID:**\
+Eine beliebige Bezeichnung für diese Applikation, die darf es aber in dem Netzwerk noch nicht geben (also z.B.: geiger_20200205)
```
Comment (ThomasWaldmann) [inline README.md:199]: Hmm, would it be reasonable to put the geiger MAC in here (not a date)?

File: README.md [hunk 10]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+Device EUI  00 D0 C0 00 C3 19 7C E8
+```
+Dann muss folgendes eingeben werden:
+
+```
+00D0C000C3197CE8
+```
+Dies gilt genauso ebenso für die **APPEUI** und den **APPKEY** .
+
+### Anmelden bei sensor.community(ehem. luftdaten.info)
```
Comment (ThomasWaldmann) [inline README.md:241]: add a blank before `(`.

File: README.md [hunk 11]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+Wenn der Multigeiger seine Daten über TTN auch an *sensor.community* weitergeben, soll muss er dort angemeldet werden.
```
Comment (ThomasWaldmann) [inline README.md:242]: shift the comma to the right position.

File: README.md [hunk 12]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+Die Anmeldung ist ziemlich so, wie oben schon beschrieben. Hier werden nur die Änderungen erläutert (gesamten Ablauf bitte oben nachlesen).
```
Comment (ThomasWaldmann) [inline README.md:243]: ... ist ziemlich ähnlich, wie ...

File: README.md [hunk 13]
```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
...
+  decoded.sw_version="" + (bytes[7]>>4) + "." + minor + "." + (bytes[8]&0xF);
+  }
+  if (port === 2) {
+    decoded.temp = ((bytes[0] * 256 + bytes[1]) / 10) + "°C";
+    decoded.humi = bytes[2] / 2 + "%";
+    decoded.press = ((bytes[3] * 256 + bytes[4]) / 10) + "hPa";
+  }
+  return decoded;
+}
+```
```
Comment (ThomasWaldmann) [inline README.md:298]: Overall, this README is way too long and it's getting worse (longer) all the time.

There is an open ticket about writing real docs: #52 and to solve the language issue #168 - this should be done soon.

File: README.md (hunk 1 above)
Comment (rexfue) [reply README.md:171]: The other method are the two spaces - and you don't like that either

File: README.md (hunk 9 above)
Comment (rexfue) [reply README.md:199]: You can put in what you want.

File: README.md (hunk 13 above)
Comment (rexfue) [reply README.md:298]: I know. But I don't want to translate all these stuff to english. If someone else is willig to do this, please do it.

File: README.md (hunk 1 above)
Comment (ThomasWaldmann) [reply README.md:171]: right. guess we just need more .rst docs and less markdown README. :-)

File: README.md (hunk 9 above)
Comment (ThomasWaldmann) [reply README.md:199]: yeah, but if you use the date, 2 users doing that at same date will collide.

File: README.md (hunk 13 above)
Comment (ThomasWaldmann) [reply README.md:298]: if the language ticket is solved first, you can write/update in german.

File: README.md (hunk 6 above)
Comment (ThomasWaldmann) [reply README.md:190]: the typo is still there.

**Pull request merged**